src/NativeTypes.dfy

```diff
@@ -0,0 +1,39 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+module NativeTypes {
+  const BASE_0:   int := 1
+
+  const BASE_1:   int := 2
+  const BASE_2:   int := 4
+  const BASE_4:   int := 16
+  const BASE_5:   int := 32
+  const BASE_8:   int := 0x1_00
+  const BASE_16:  int := 0x1_0000
+  const BASE_24:  int := 0x1_000000
+  const BASE_32:  int := 0x1_00000000
+  const BASE_40:  int := 0x1_00000000_00
```

Comment:
Wouldn't it be more natural to write this as `0x100_00000000` (and similar for the others)?